### DISCUSSION TITLE
Unable to load the mapper

### DISCUSSION THREAD

**Discussion by karthikjeyaram** [atlasmap/atlasmap#3936] [Q&A]
Dear Community,

When opening the mapper UI, getting the below error. Unable to resolve even after restarts. Can someone please advise how to fix this?

**_"Unable to decompress the aggregate mappings catalog buffer"_**

Thank you
![mapper-error](https://user-images.githubusercontent.com/102969288/162869324-254b5d82-6808-4ec3-94d3-e6c70db7f60e.JPG)
.

**Comment by igarashitm** [comment]
It sounds like the ADM file is broken. Do you see anything in the backend console? Also can you upload the ADM file you are trying to import?

**Reply by igarashitm** [reply]
And which version of AtlasMap UI are you using?

**Reply by karthikjeyaram** [reply]
> And which version of AtlasMap UI are you using?

We are using 2.1.6.

**Reply by karthikjeyaram** [reply]
> It sounds like the ADM file is broken. Do you see anything in the backend console? Also can you upload the ADM file you are trying to import?
[RSSFEED-MAPPING (2).zip](https://github.com/atlasmap/atlasmap/files/8473116/RSSFEED-MAPPING.2.zip)

please find attached adm files

**Reply by karthikjeyaram** [reply]
Will these .adm and other mapping related files stored in any temp location or cache? Even after removing the suspected mapping files from the respective file store, the issue still persists.. any suggestions here?

**Reply by igarashitm** [reply]
Can you try with the latest 2.3.18?

**Reply by karthikjeyaram** [reply]
ok sure. we are already in the process of moving to this version, but there are some compatibility issue in our existing setup. so this would take sometime. meanwhile, if there is a workaround or suggestion to fix this, that would be really great..

**Reply by karthikjeyaram** [reply]
This issue is resolved after we redeployed the AtlasMap component. Thank you.
